### ISSUE TITLE
[CoE Starter Kit - QUESTION] Audit Log Setup Troubleshooting - "New Connection" button does not work

### ISSUE THREAD

**Issue by firefox-edge123** [microsoft/coe-starter-kit#5381]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

I previously tried setting up the Audit Log with the Dec 2022 release, but couldn't get it to work.
I kept getting the 401 error message.
![401](https://user-images.githubusercontent.com/128085471/234396677-02f1f4b8-e83d-4dfc-a025-1ec4a32b8d2d.PNG)

I then upgraded to the April 2023 release and I still got the 401 error message.

I then attempted to follow the instructions here:
https://github.com/microsoft/coe-starter-kit/issues/4961

But at Step 5, when I click the "New connection" button and sign into my Global Admin account, the "Selected connection" remains greyed out.
![New-Connection-Greyed-Out](https://user-images.githubusercontent.com/128085471/234397086-5c5968ae-5f5c-4dab-b10d-d661aad8c006.PNG)

I click the "refresh connections" button and no change.

I went back into the "Security" tab and made sure I had the correct Identity Provider (Azure Active Directory), Client ID, Client secret, Authorization URL, and Resource URL.

I kept Tenant ID as the default (common).
![Audit-Log-Troubleshooting](https://user-images.githubusercontent.com/128085471/234397196-66b11401-b6b6-46ba-ad29-4b7c88946bd7.png)

Then clicked "Update connector".
I tried testing again, but I still get the same thing:
I click the "New connection" button and sign into my Global Admin account, the "Selected connection" remains greyed out.

What would the next troubleshooting step be?

### What solution are you experiencing the issue with?

None

### What solution version are you using?

April 2023

### What method are you using to get inventory and telemetry?

None

**Comment by Jenefer-Monroe** [comment]
Sometimes the product doesn't let you create the connection there for some reason. 
You can create it in the connections page instead
![image](https://user-images.githubusercontent.com/89584709/234403418-ab7d7b46-2de3-421d-bcb3-a50feddd339b.png)
![image](https://user-images.githubusercontent.com/89584709/234403571-ea904ef2-da2c-4995-88f3-f0d61ff7072a.png)

**Comment by firefox-edge123** [comment]
Hi Jenefer,

Your suggestion helped me troubleshoot and get my Audit Log setup successfully. Thank you!

I followed your suggestion and got an error which made me realize I made a silly mistake:
I put the "Secret ID" instead of the "Secret Value" in the "Client Secret" field.

After I put the "Secret Value" in the "Client Secret" field, and clicked the "Update Connector" button, I was able to setup the Audit log successfully. 

Thank you so much for your help.
